### DISCUSSION TITLE
Make Instructions for Creating an Executable Publicly Available

### DISCUSSION THREAD

**Discussion by Zach-Attach** [buildingamind/NewbornEmbodiedTuringTest#37] [Ideas]
In the README.md, [the first step for "Running a NETT"](https://github.com/buildingamind/NewbornEmbodiedTuringTest/blob/49e163b225fe76cbe42e7a3ef94506098da0793c/README.md?plain=1#L65) currently does not have linked written or video instructions for creating an executable.

**Comment by desaibhargav** [comment]
Hi @Zach-Attach, creating an executable would require access to the Unity codebase, which we want to keep locked as of now. Once we have the executables hosted, I will update the README.md to the link to the page where users can download them from.

**Comment by Zach-Attach** [comment]
Splitting into 2 issues:
1. (this one) will be for creating executables (version 0.3 or later)
2. linking to existing website for hosting premade executables (version 0.2)  
        https://github.com/buildingamind/NewbornEmbodiedTuringTest/issues/32
